### ISSUE TITLE
Suggested Mockup for combination of Awesome Swift and Open-Source App

### ISSUE THREAD

**Issue by michieriffic** [matteocrippa/awesomeSwift-iOS-App#3]
@matteocrippa and @dkhamsing

Here is the suggested Mockup for the combination of Awesome resources and Open-Source iOS App: 

![forawesomeswiftandopensourceios](https://cloud.githubusercontent.com/assets/12575688/17121255/fbfa05ec-5304-11e6-928a-7bc0f7ba86dd.png)

**Comment by dkhamsing** [comment]
Nice

**Comment by matteocrippa** [comment]
What do you think to remove lines even in ProjectDetailVC for the Github repo, app store, etc links?

**Comment by michieriffic** [comment]
@matteocrippa Yeah I think without the lines look better. I have updated mockup to have a sample of no lines look on the ProjectDetailVC under the without line section :)

**Comment by matteocrippa** [comment]
I'm on holiday these days.. so I have some spare time to work on this project.

I noticed that @michieriffic omitted the favorite tab, is this done by purpose? Or can I assume to put it as the first tab item?

So we will have:

- Favorite;
- Swift;
- OSA;
- Credits;

**Comment by dkhamsing** [comment]
Hm maybe 

1. Swift
- OSA
- Favorites
- Credits

Since Favorites could be empty

**Comment by matteocrippa** [comment]
@dkhamsing that sorting was my first idea, but I started thinking about the normal usage, is repos more relevant or favorites? btw within the new push that I plan to do tonight tab bar sorting will not be so hard to manage

**Comment by michieriffic** [comment]
@matteocrippa I didn't omit it on purpose. I just forgotten about it. LOL :)
